pkg/hardware/host/gpu.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package host
+
+import (
+	"log"
+
+	"github.com/microsoft/wmi/pkg/base/host"
+	"github.com/microsoft/wmi/pkg/base/instance"
+	"github.com/microsoft/wmi/pkg/base/query"
+	"github.com/microsoft/wmi/pkg/constant"
+	"github.com/microsoft/wmi/pkg/virtualization/core/gpu"
+)
+
+// GetPartitionableGpuCollection gets all host partitionable GPUs
+func GetPartitionableGpuCollection(whost *host.WmiHost) (partitionablegpucollection gpu.PartitionableGpuCollection, err error) {
```

Comment:
Since the method gets GetWmiInstancesFromHost, I thought it would be better to have this method under hardware/host.

But I also see your point that since we are returning the collection object, which is a virtualization concept, so it is better to have it under virtualization/core/gpu.